source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCard.mm

```diff
@@ -111,4 +111,24 @@ - (void)setCard:(std::shared_ptr<AdaptiveCard> const &)card
     return nil;
 }
 
+- (NSData *)additionalProperty
+{
+    if (_adaptiveCard) {
+        Json::Value blob = _adaptiveCard->GetAdditionalProperties();
+        if (blob.empty()) {
+            return nil;
+        }
+        Json::StreamWriterBuilder streamWriterBuilder;
+        auto writer = streamWriterBuilder.newStreamWriter();
+        std::stringstream sstream;
+        writer->write(blob, &sstream);
+        delete writer;
```

Comment:
If you use `std::unique_ptr<>`, its lifetime is automatically managed for you :)

```suggestion
        std::unique_ptr<StreamWriter> writer(streamWriterBuilder.newStreamWriter());
        std::stringstream sstream;
        writer->write(blob, &sstream);
```